### PR TITLE
Using sagas for async request - added a simple beer-example

### PR DESCRIPTION
**Pull request by hessenh**
Fetch beerlist and display a list. 
- Sagas to manage async calls with redux
- Superagent for actual call

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ module.exports = {
   context: path.resolve(__dirname, './src'),
   target: 'web',
   entry: [
+    'babel-polyfill',
```
Comment (hawkaa) [inline webpack.config.js]: Why is this needed?

File: src/configureStore.js [hunk 2]
```diff
@@ -1,11 +1,19 @@
 import { createStore, applyMiddleware } from 'redux';
+import createSagaMiddleware from 'redux-saga';
 import createLogger from 'redux-logger';
 
 import rootReducer from './reducers/';
+import sagas from './sagas/index';
```
Comment (johnolos) [inline src/configureStore.js:7]: Sagas might fit better into the world of redux which we already are engaged to, but I'm going to ask anyway.
Did you consider [axios](https://github.com/mzabriskie/axios)?

**Review by johnolos — COMMENTED** [review]
Looks very neat. :+1:

File: webpack.config.js (hunk 1 above)
Comment (hessenh) [reply webpack.config.js]: Got the error described here: http://mysticcoders.com/blog/fixing-regenerator-runtime-error-integrating-redux-saga
Now importing it where it should be. Thanks!

File: src/configureStore.js (hunk 2 above)
Comment (hessenh) [reply src/configureStore.js:7]: Hm.. I think this explains the redux-saga concept better than I do: 
"The mental model is that a saga is like a separate thread in your application that's solely responsible for side effects. redux-saga is a redux middleware, which means this thread can be started, paused and cancelled from the main application with normal redux actions, it has access to the full redux application state and it can dispatch redux actions as well." 

I think axios is more comparable to superagent (https://github.com/Pubcrawler/weiss/blob/1c775b72c01465854c6a8e8adde12e613ae8484f/src/services/index.js). We could easily use axios :)

File: webpack.production.config.js [hunk 3]
```diff
@@ -5,7 +5,10 @@ const nodeExternals = require('webpack-node-externals');
 module.exports = [{
   context: path.resolve(__dirname, './src'),
   target: 'web',
-  entry: './index.jsx',
+  entry: [
+    'babel-polyfill',
```
Comment (hessenh) [inline webpack.production.config.js]: Note to self. Remove this.

**Review by magnumog — APPROVED** [review]
Looks good, good for learning some easy stuff to begin with with counter and beer.
